### PR TITLE
Moving from Elasticsearch to OpenSearch

### PR DESCRIPTION
**Pull request by psaiz**
:heart: Thank you for your contribution!

### Description

This closes https://github.com/inveniosoftware/docs-invenio-rdm/issues/432
It renames Elasticsearch to OpenSearch and changes the link to the documentation.

### Checklist

Ticks in all boxes and 🟢 on all GitHub actions status checks are required to merge:

- [x] I'm aware of the [code of conduct](https://inveniordm.docs.cern.ch/contribute/code-of-conduct/).
- [x] I've created [logical separate commits](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#commits) and followed the [commit message format](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#commit-message).
- [x] I've added relevant test cases.
- [x] I've added relevant documentation.
- [x] I've marked [translation strings](https://inveniordm.docs.cern.ch/develop/best-practices/i18n/) (for relevant code).
- [x] I've followed the [CSS/JS](https://inveniordm.docs.cern.ch/develop/best-practices/css-js/) and [React](https://inveniordm.docs.cern.ch/develop/best-practices/react/) guidelines (for relevant code).
- [x] I've followed the [web accessibility](https://inveniordm.docs.cern.ch/develop/best-practices/accessibility/) guidelines (for relevant code).
- [x] I've followed the [user interface](https://inveniordm.docs.cern.ch/develop/best-practices/ui/) guidelines (for relevant code).
- [x] I've identified the [copyright holder(s)](https://inveniordm.docs.cern.ch/contribute/copyright-policy/) and updated copyright headers for touched files (>15 lines contributions).
- [x] I've NOT included third-party code (copy/pasted source code or new dependencies).

**Third-party code**

If you've added [third-party code (copy/pasted or new dependencies)](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#third-party-codedependencies), please reach out to an [architect](https://github.com/orgs/inveniosoftware/teams/architects/members).

**Reminder**

By using GitHub, you have already agreed to the [GitHub’s Terms of Service](https://help.github.com/articles/github-terms-of-service/#6-contributions-under-repository-license) including that:

1. You license your contribution under the same terms as the current repository’s license.
2. You agree that you have the right to license your contribution under the current repository’s license.

### CODE REVIEW

File: docs/install/requirements.md [hunk 1]
```diff
@@ -97,10 +97,10 @@ and adjust the `Memory` to the corresponding value. If you have a few cores
 more to spare, it might be a good idea to give more than 2. Take into account
 that you will run between 4 and 8 containers.
 
-#### Elasticsearch and Docker (macOS and Linux)
+#### OpenSearch and Docker (macOS and Linux)
 
-Among the containers you will run is an Elasticsearch container which is quite demanding.
-Per Elasticsearch's [Docker documentation](https://www.elastic.co/guide/en/elasticsearch/reference/7.9/docker.html#docker-prod-prerequisites),
+Among the containers you will run is an OpenSearch container which is quite demanding.
+Per OpenSearch's [Docker documentation]([https://www.elastic.co/guide/en/elasticsearch/reference/7.9/docker.html#docker-prod-prerequisites](https://opensearch.org/docs/latest/install-and-configure/install-opensearch/docker/)),
```
Comment (fenekku) [inline docs/install/requirements.md:103]: Oups bad copy-paste here!

**Review by fenekku — APPROVED** [review]
Approved pending fix to markdown.

File: docs/install/requirements.md (hunk 1 above)
Comment (psaiz) [reply docs/install/requirements.md:103]:
```suggestion
Per OpenSearch's [Docker documentation](https://opensearch.org/docs/latest/install-and-configure/install-opensearch/docker/)),
```

Comment (psaiz) [reply docs/install/requirements.md:103]: Thanks for catching that!

**Comment by psaiz** [comment]
Thanks @fenekku ! Moving it to approved then

**Review by zzacharo — APPROVED** [review]

**Pull request merged**